README.md

```diff
@@ -33,6 +33,7 @@ If you are developing a project that uses new CSS language features and must wor
 - Enables [postcss-import](https://github.com/postcss/postcss-import) so that `@import` statements are inlined
 - Optionally enables [postcss-url](https://github.com/postcss/postcss-url) so that `url()` statements are processed
 - Enables [postcss-mixins](https://github.com/postcss/postcss-mixins) so that you can define mixins
+- Enables [postcss-calc](https://github.com/postcss/postcss-calc) Lets you reduce calc() references whenever it's possible
```

Comment:
can we change the sentence to something like: 

> Enables [postcss-calc](https://github.com/postcss/postcss-calc) so that `calc()` references are reduced whenever it's possible

Reply:
agree! done 👍